x-pack/plugins/ingest_manager/server/saved_objects/index.ts

```diff
@@ -65,9 +65,9 @@ const savedObjectTypes: { [key: string]: SavedObjectsType } = {
         config_revision: { type: 'integer' },
         config_newest_revision: { type: 'integer' },
         default_api_key_id: { type: 'keyword' },
-        default_api_key: { type: 'keyword' },
+        default_api_key: { type: 'binary', index: false },
```

Comment:
binary? I assume this is something SO specific.

Reply:
Yes the doc for encrypted saved object, recommend to put encrypted field as binary